website/src/pages/index.tsx

```diff
@@ -54,7 +54,7 @@ function HeroBanner() {
             <Translate>Get Started</Translate>
           </Link>
           <Link className="button button--info" to="https://docusaurus.new">
-            <Translate>Playground</Translate>
+            <Translate>Try a Demo</Translate>
```

Comment:
The verb is probably more appropriate as label of this button, isn't it?

Reply:
Yes, I think "Try a demo" may be more straightforward than "playground" for non-tech users.